src/aat/resources/features/F-102/S-580.td.json

```diff
@@ -508,7 +508,7 @@
 									"pre_states": [
 										"IN_PROGRESS"
 									],
-									"post_state": null,
+									"post_states": [],
```

Comment:
it would have been useful to have one example with a non empty array?

Reply:
we can add the non-empty array example during the functional testing part of this item